packages/cloud/src/routes/tenants.ts

```diff
@@ -9,6 +9,37 @@ export const tenantsRoutes = (library: TenantsLibrary) =>
     .get('/', { response: tenantInfoGuard.array() }, async (context, next) => {
       return next({ ...context, json: await library.getAvailableTenants(context.auth.id) });
     })
+    .patch(
+      '/:tenantId',
+      {
+        body: createTenantGuard.pick({ name: true, tag: true }).partial(),
+        response: tenantInfoGuard,
+      },
+      async (context, next) => {
+        const availableTenants = await library.getAvailableTenants(context.auth.id);
+        if (
+          /**
+           * Authenticated user with `ManageTenant` scope can update any user tenant;
+           * authenticated user with `ManageTenantSelf` scope can only update their own tenant.
```

Comment:
Align the capitalization?   Besides, feels like the comments could leave at the enum scope definition place.